django_project/feti/api_views/address.py

```diff
@@ -0,0 +1,25 @@
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from feti.models.address import Address
+
+from feti.serializers.address_serializer import AddressSerializer
+
+__author__ = 'Alison Mukoma <alison@kartoza.com>'
+__date__ = '22/10/17'
+__license__ = "GPL"
+__copyright__ = 'kartoza.com'
+
+
+class AddressApiView(APIView):
```

Comment:
Returning only all addresses are meaningless, we don't know whose address is this